File: SEP3_T2/grpc-springboot/src/main/java/sep3/model/SocialMediaUser.java
package sep3.model;

public class SocialMediaUser extends User
{
    public int followersCount;
    public int followingCount;
    public String email;

    public SocialMediaUser(String username, String password, String email)
    {
        super(username, password);
        this.email = email;
        this.followersCount = 0;
        this.followingCount = 0;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }
}

File: SEP3_T1/Server/Entities/SocialMediaUser.cs
namespace Entities;

public class SocialMediaUser
{
    public string Username { get; set; }
    public string Password { get; set; }
    public string Email { get; set; }
    
    public SocialMediaUser()
    {
        
    }
    
    public SocialMediaUser(string username, string password, string email)
    {
        Username = username;
        Password = password;
        Email = email;
    }
}